### ISSUE TITLE
Back button link should be a link

### ISSUE THREAD

**Issue by dsamojlenko** [cds-snc/c19-benefits-node#212]
The “Back” button should either be an `<a>`, or should have role=”link”.

**Comment by katedee** [comment]
I would add role link, because it doesn't actually have an HREF. It _IS_ actually a button, and when you click it, it triggers a function to navigate back
